Sources/MongoSwift/ChangeStream.swift

```diff
@@ -90,60 +105,101 @@ public class ChangeStream<T: Codable>: Sequence, IteratorProtocol {
         return extractMongoError(error: error)
     }
 
+    /// Poll the underlying mongoc_change_stream_t for the next change event.
+    internal func fetchNextDocument() throws -> T? {
```

Comment:
this is similar to `getNextDocumentFromMongocCursor`, but I went with a shorter and more generic name.